lib/services/common-api-presenter.js

```diff
@@ -316,6 +299,36 @@ function CommonApiPresenterFactory(
             });
         };
 
+        CommonApiPresenter.prototype._buildContext = function(graphContext) {
+            var self = this;
+            var scope = self.res.locals.scope;
+            var config = configuration.getAll();
```

Comment:
@zyoung51 It may make sense to cache this value as a class property on startup, as I found it can be quite an expensive call at runtime (blocking CPU for ~40ms, see my comment here: https://github.com/RackHD/on-tasks/blob/master/lib/task.js#L387-L390). Alone, it's not so bad, but when we have a bunch of pxe clients triggering property rendering operations all at the same time, it causes the system to be pretty laggy exactly when we don't want it to be.